service/history/ndc/activity_state_replicator.go

```diff
@@ -195,23 +323,9 @@ func (r *ActivityStateReplicatorImpl) SyncActivityState(
 	if _, err := workflow.NewTimerSequence(
 		mutableState,
 	).CreateNextActivityTimer(); err != nil {
```

Comment:
this function should be invoked if 
* there is >= 1 activity updated
* just before updating mutable state?